Examples/DemoPlacementViewController.swift

```diff
@@ -80,17 +83,17 @@ class DemoPlacementViewController: UIViewController {
 
     private func defaultMaterials() -> [SCNMaterial] {
         let groundImage = SCNMaterial()
-        groundImage.diffuse.contents = UIColor.darkGray
+        groundImage.diffuse.contents = UIColor.darkGray.withAlphaComponent(0.5)
```

Comment:
Any reason not to use  the rgba init instead? (https://developer.apple.com/documentation/uikit/uicolor/1621925-init)

Reply:
I think this is just an easy way set alpha when using the UIColor presets. Ultimately is does end up setting the rgba value from your link.